openghg/standardise/column/_openghg.py

```diff
@@ -260,6 +260,7 @@ def parse_openghg(
     # Update attributes to match metadata after cleaning
     attributes.update(metadata)
 
+    # TODO: Decide if the key here should be more descriptive that just `species`
```

Comment:
Yeah I think we can do away with this nested dictionary returning when we refactor the assignment of data to the Datasources. I think we'll be able to simplify that section of code quite a bit.